client/src/main/java/com/paypal/selion/internal/platform/grid/MobileTestSession.java

```diff
@@ -212,14 +224,21 @@ public void initializeTestSession(InvokedMethodInformation method) {
         }
 
         boolean appPathProvided = StringUtils.isNotBlank(appPath);
+        
+        //App run and browser run should be mutually exclusive. If appName or appPath is provided disregard
+        //the browser parameter
+        
+        if(StringUtils.isNotBlank(appName) || StringUtils.isNotBlank(appPath)){
```

Comment:
slight nit pick here... 
please fix the whitespace so that it is consistent with the rest of this file.